src/test/java/GoServletTest.java

```diff
@@ -0,0 +1,143 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import com.google.api.services.tasks.model.Task;
+import com.google.common.collect.ImmutableList;
+import com.google.sps.model.DirectionsClientFactory;
+import com.google.sps.model.TasksClientFactory;
+import com.google.sps.servlets.GoServlet;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+
+@RunWith(JUnit4.class)
+public class GoServletTest extends AuthenticatedServletTestBase {
```

Comment:
The servlet test should be testing that the response is correctly created given some input from the client. 

Currently, while this does a good job at testing the `getLocations` method, this does not do this. You likely need more tests here for the servlet responses themselves.

Reply:
You're absolutely right about this. I think the approach I went with to make sure the Go servlet PR isn't overly large does not take testing into consideration as much as it should. Now that I think of it - it's probably better to just merge this PR with one that passes the addresses into Directions and returns the result to the client so that on top of being able to test the servlet as a whole, a working MVP is available as well. The Places API can be an "add-on" for later.

Reply:
This doesn't apply anymore because I have moved the `getLocations` logic into a utility class of its own after addressing the comment directly above this. Thanks!

Reply:
`RESOLVED`